v2/internal/runner/enumerate.go

```diff
@@ -22,8 +23,39 @@ func (r *Runner) runStandardEnumeration(executerOpts protocols.ExecuterOptions,
 	return r.executeTemplatesInput(store, engine)
 }
 
+// Get all the scan lists for a user/apikey.
+func (r *Runner) getScanList() error {
+	client := nucleicloud.New(r.options.CloudURL, r.options.CloudAPIKey)
```

Comment:
the client should be created globally and reused for all functions instead of creating a new client each time as being done here and in the below methods

Reply:
Yes, this makes sense.